htdocs/assets/js/util.js

```diff
@@ -4,15 +4,27 @@ define(function(require) {
         _ = require('underscore'),
         URI = require('uri'),
         Autosize = require('autosize'),
-        CodeMirror = require('codemirror');
+        CodeMirror = require('codemirror'),
+        Moment = require('moment'),
+        Data = require('data');
+        // loads tz data
+        require('moment-timezone');
```

Comment:
That seems aight. I just want to have The World set up by the time the app object is initialized.